.gitignore

```diff
@@ -2,4 +2,5 @@ _site/
 node_modules
 .firebase/
 data-source/
-addresses.json
\ No newline at end of file
+addresses.json
+.DS_Store
```

Comment:
I'm not seeing this in my hidden folder list. Is this necessary?

Reply:
@dwightjl I can remove if you'd prefer.

Reply:
LGTM thanks for clarifying!